Gemfile

```diff
@@ -15,8 +15,7 @@ gem "cocoon"
 gem "devise"
 gem "devise_invitable"
 gem "dotenv-rails"
-gem "flipper"
-gem "flipper-active_record"
+gem "flipper", git: "https://github.com/jnunemaker/flipper.git", tag: "v0.16.2"
```

Comment:
What's the reason behind this? Is the mainline `flipper` gem not Rails 6 compatible?